src/Integration.Vsix.UnitTests/CFamily/IntegrationTests/CFamily_CLangAnalyzer_IntegrationTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
```

Comment:
Do we have a rule that could remind us to add copyright notices? ;-)

Reply:
If only!